### PR TITLE
KubeCon NA 2018 talk

### PR DESCRIPTION
**Pull request by hjacobs**
Add Alexander's talk (https://kccna18.sched.com/event/GrU0).

### CODE REVIEW

**Comment by RafiaSabih** [comment]
:+1:

File: docs/index.md [hunk 1]
```diff
@@ -51,10 +51,12 @@ Please, report any issues discovered to https://github.com/zalando/postgres-oper
 
 ## Talks
 
-1. "PostgreSQL and Kubernetes: DBaaS without a vendor-lock" talk by Oleksii Kliukin, PostgreSQL Sessions 2018: [video](https://www.youtube.com/watch?v=q26U2rQcqMw) | [slides](https://speakerdeck.com/alexeyklyukin/postgresql-and-kubernetes-dbaas-without-a-vendor-lock)
+1. "Building your own PostgreSQL-as-a-Service on Kubernetes" talk by Alexander Kukushkin: [video](https://www.youtube.com/watch?v=G8MnpkbhClc) | [slides](https://static.sched.com/hosted_files/kccna18/1d/Building%20your%20own%20PostgreSQL-as-a-Service%20on%20Kubernetes.pdf)
```
Comment (FxKu) [inline docs/index.md]: Please, add the conference and year like in the other listed talks.

```suggestion
1. "Building your own PostgreSQL-as-a-Service on Kubernetes" talk by Alexander Kukushkin, KubeCon NA 2018: [video](https://www.youtube.com/watch?v=G8MnpkbhClc) | [slides](https://static.sched.com/hosted_files/kccna18/1d/Building%20your%20own%20PostgreSQL-as-a-Service%20on%20Kubernetes.pdf)
```

**Comment by hjacobs** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Pull request merged**